.circleci/config.yml

```diff
@@ -103,12 +110,38 @@ jobs:
       - attach_workspace:
           at: /home/circleci
       - installation-tests
+      - persist_to_workspace:
+          root: /home/circleci/
+          paths:
+            - project/CIVENV
+    environment:
+      CI_VENV_NAME: CIVENV
+    
   
   integration-test:
     docker:
       - image: sematicai/sematic-ci:latest
     steps:
-      - run: echo "no op for now"
+      - attach_workspace:
+          at: /home/circleci
+      - run: 
+          name: Create a run
+          command: source ./CIVENV/bin/activate && sematic start && sematic run examples/add
```

Comment:
Optionally, we can run this in a dedicated script. But laying them here is more readable.